### ISSUE TITLE
Adding new files need watch to be restarted

### ISSUE THREAD

**Issue by robotlabs** [rehabstudio/fe-skeleton#36]
Gulp Watch need to be restarted in order to enjoy the new files added. Would be nice to avoid this

**Comment by njmcode** [comment]
Is this still an issue?

**Comment by antwan1986** [comment]
Native gulp watch doesn't pick up files added to the filesystem after it
was initialised. It's a limitation I've accepted but there's certainly node
modules out there which poll for changes. Just needs investigated.

On Friday, 27 November 2015, Neil McCallion <notifications@github.com>
wrote:

> Is this still an issue?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rehabstudio/fe-skeleton/issues/36#issuecomment-160116338>
> .
>


-- 

*Anthony Hastings*  //  Senior Front End Developer

*+rehabstudio <https://www.rehabstudio.com>* - Making it better.

**Comment by antwan1986** [comment]
I did some investigation on the issue this morning. In gulp `4.0.0` there is a new file watcher under the hood called [chokidar](https://www.npmjs.com/package/chokidar) which has the ability to watch new files, and realise when others have been deleted. I discerned all this information from the [closed issue](https://github.com/gulpjs/gulp/issues/651) on gulpjs repository. There's nothing to action here except patiently wait for the release of `4.0.0`.

Huzzah!
